packages/core/src/agent/index.ts

```diff
@@ -546,12 +546,32 @@ export class Agent<
   async generateTitleFromUserMessage({
     message,
     runtimeContext = new RuntimeContext(),
+    model,
   }: {
     message: string | MessageInput;
     runtimeContext?: RuntimeContext;
+    model?: MastraLanguageModel | ((ctx: RuntimeContext) => MastraLanguageModel | Promise<MastraLanguageModel>);
```

Comment:
You can simplify this definition as follows.
Refs: https://github.com/mastra-ai/mastra/pull/5472/files#r2168111254

```suggestion
    model?: DynamicArgument<MastraLanguageModel>;
```